### ISSUE TITLE
Add unit tests for vitalRates.R function

### ISSUE THREAD

**Issue by jonesor** [jonesor/Rage#110]
https://codecov.io/gh/jonesor/Rage/src/master/R/vitalRates.R

**Comment by jonesor** [comment]
Done.

**Comment by jonesor** [comment]
The unit tests appear to break the package, so I've removed them to try and figure out what the problem is.
The funny thing is that the build check works on my local machine, but then fails on Travis, Github and Appveyor.

The error thrown is:
`error: 'length(x) = 4 > 1' in coercion to 'logical(1)' `

Research shows that this is a common problem.

Should set environment variable to replicate the problem on local machine.
`Sys.setenv("_R_CHECK_LENGTH_1_CONDITION_" = "true")`

Current status is that I've removed the unit tests for `vitalRates` and will try to find what causes it.
